### PR TITLE
Add post for Moore annual report

### PR DESCRIPTION
**Pull request by ErinBecker**
@tracykteal Please look over and make edits. Planning to post on Monday if everything looks ok.

### CODE REVIEW

**Comment by ErinBecker** [comment]
@tracykteal Can we move this post to today? Naupaka's Data in the Field post will be going up on Monday and I'd like to leave Mondays clear for that series if possible. Please let me know if this looks ready to  you.

File: _posts/2016-12-05-moore-report.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- [Output and Events - Excel file](https://github.com/datacarpentry/practices/blob/datacarpentry/datacarpentry/Data-Carpentry_Practices-Data-Collection.xlsx?raw=true)  
+
+
+
+### Some highlights from the last year  
+
+#### Workshops  
+
+- Since August, 2015 we have run 58 workshops in ten countries, serving approximately 1,100 learners  
+- We have over 600 completed surveys that show the demographics of our audience as well as workshop outcomes.
```
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: We've now run 93 workshops for ~1700 learners. We can update the numbers, or put Aug 2015 to Sept 2016.

File: _posts/2016-12-05-moore-report.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+show_meta: true
+authors: ["Tracy Teal"]
+---
+
+
+A year ago we were fortunate enough to receive a grant from the 
+[Gordon and Betty Moore Foundation](https://www.moore.org/article-detail?newsUrlName=the-gordon-and-betty-moore-foundation-selects-awardees-for-$21-million-in-grants-to-stimulate-data-driven-discovery) 
+Data-Driven Discovery initiative to support Data Carpentry to develop and deliver data skills training to researchers in the life and 
+physical sciences. [See our original proposal here.](https://figshare.com/articles/Data_Carpentry_Moore_Foundation_Grant_Proposal/1507552)
+Thanks to the Moore Foundation, this support has allowed us to run more workshops, develop new materials,
```
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: allowed us -> allowed us as a community

File: _posts/2016-12-05-moore-report.md (hunk 1 above)
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: we have run -> there have been

File: _posts/2016-12-05-moore-report.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- Over 50% of learners were new to programming having never programmed or less than once per year.  
+- The vast majority (>90%) of those responding to our post-workshop survey say that participating in the 
+workshop was worth their time and led to improvements in their data management and data analysis skills.  
+- Gender balance in USA workshops (where demographic data is collected) was 51% female and 47% male.  
+- In the USA we are primarily reaching Caucasian (63%) and Asian/Pacific Islander (18%) participants. Black or African American
+(3%), Hispanic (6%) and American or Alaskan Native (0.2%) are under-represented.  
+
+#### Instructors  
+- With Software Carpentry we have run over 30 instructor training events.  
+- We have over 200 officially badged Data Carpentry instructors.
```
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: There are over 350 officially badged DC instructors

File: _posts/2016-12-05-moore-report.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+#### Curriculum  
+- With [CyVerse](http://www.cyverse.org/) and [SESYNC](https://www.sesync.org/) we developed a
+[Genomics Data Carpentry](http://www.datacarpentry.org/genomics-workshop/) workshop, with a grant from Amazon AWS for cloud computing
+resources  
+- With [NEON](http://www.neonscience.org/) we developed Geospatial Data Carpentry lessons on 
+[working with raster data](http://neondataskills.org/tutorial-series/raster-data-series/) and 
+[vector data](http://neondataskills.org/tutorial-series/vector-data-series/).  
+
+#### Strategic Planning, Sustainability, Communication and Advocacy  
+- Hired our Program Coordinator, [Associate Director](http://www.datacarpentry.org/blog/new-associate-director/)
```
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: Hired our -> Hired a

File: _posts/2016-12-05-moore-report.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+- Developed criteria for [requirements for Data Carpentry workshops](http://www.datacarpentry.org/workshops-host/) and 
+[self-organized](http://www.datacarpentry.org/self-organized-workshops/) workshops.  
+- Gave [presentations and wrote papers](https://www.zotero.org/groups/datacarpentry) on Data Carpentry and data skills for more 
+effective and reproducible research.  
+
+
+In year one we focused on building the foundation to scale to run more workshops and train instructors, develop materials in new domains,
+maintain existing lessons and support our instructor community. We also are working to establish Data Carpentry with Software Carpentry
+as sustainable organizations. We'll continue these activities in year two and and also work on scaling to reach more learners throughout
+the world!
```
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: [suggestion] reach and engage with more learners and instructors throughout ..

File: _posts/2016-12-05-moore-report.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+[Gordon and Betty Moore Foundation](https://www.moore.org/article-detail?newsUrlName=the-gordon-and-betty-moore-foundation-selects-awardees-for-$21-million-in-grants-to-stimulate-data-driven-discovery) 
+Data-Driven Discovery initiative to support Data Carpentry to develop and deliver data skills training to researchers in the life and 
+physical sciences. [See our original proposal here.](https://figshare.com/articles/Data_Carpentry_Moore_Foundation_Grant_Proposal/1507552)
+Thanks to the Moore Foundation, this support has allowed us to run more workshops, develop new materials, 
+develop infrastructure and advocate for the importance of data skills training for more effective and reproducible research.  
+
+
+We wanted to share the outputs and outcomes of Data Carpentry activities over this last year from our annual report to the Moore 
+Foundation.  
+
```
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: [suggestion] The work of the community has made this possible, and we wanted to share our Moore annual report to highlight all the work that has been done in the last year!

File: _posts/2016-12-05-moore-report.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+- Developed a [Lesson Development Roadmap](http://www.datacarpentry.org/lessons-incubation/) to guide the development
+of new curriculum  
+- Started a joint [Partnership](http://www.datacarpentry.org/partnerships/) program with Software Carpentry.  
+- Developed criteria for [requirements for Data Carpentry workshops](http://www.datacarpentry.org/workshops-host/) and 
+[self-organized](http://www.datacarpentry.org/self-organized-workshops/) workshops.  
+- Gave [presentations and wrote papers](https://www.zotero.org/groups/datacarpentry) on Data Carpentry and data skills for more 
+effective and reproducible research.  
+
+
+In year one we focused on building the foundation to scale to run more workshops and train instructors, develop materials in new domains,
```
Comment (tracykteal) [inline _posts/2016-12-05-moore-report.md]: In year one the focus was on building ...

**Comment by tracykteal** [comment]
Let's put Naupaka's up today and this one can go up tomorrow.

**Comment by ErinBecker** [comment]
Sounds good.

**Pull request merged**